libs/entitlement/bridge/src/main/java/org/elasticsearch/entitlement/bridge/Util.java

```diff
@@ -23,6 +25,8 @@ public class Util {
     public static final Class<?> NO_CLASS = new Object() {
     }.getClass();
 
+    private static final Set<String> skipInternalPackages = Set.of("java.lang.invoke", "java.lang.reflect", "jdk.internal.reflect");
```

Comment:
This set leaves me a bit uneasy -- we should at least test this is complete for all JDKs we support

Reply:
This will be difficult to do ahead of time without a lot of work. I'm going to leave this up to our jdk matrix tests that are run on main, and will watch that job.